nservicebus/messaging/publish-subscribe/index.md

```diff
@@ -112,7 +112,7 @@ Transport that support native publish–subscribe
 The subscribe workflow for native transports is as follows
 
  1. Subscribers request to the transport the intent to subscribe to certain message types.
- 1. Transport stores both the subscriber names and the message types.
+ 1. Broker stores the subscription information.
```

Comment:
It's incorrect to say that the transport is the one that stores subscription info or sends a message to the subscribers. Broker is the one that does it (i.e. non-NSB part).